rules-reviewed/eap7/eap6/hibernate4-xml.windup.xml

```diff
@@ -11,8 +11,10 @@
             <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
         </dependencies>
         <sourceTechnology id="hibernate" versionRange="[4,5)" />
+        <sourceTechnology id="eap" versionRange="[6,7)" />
         <targetTechnology id="hibernate5" versionRange="[5,)" />
         <targetTechnology id="eap7" versionRange="[7,)" />
+        <targetTechnology id="eap" versionRange="[7,)" />
```

Comment:
true, but we will need to add eap7 metadata too, so after succesfull experimenting with eap, I will change/remove it. ;)